### PR TITLE
[Frontend] Integrating payment interface with the app

### LINKED ISSUE #90: [Frontend] Integrating payment interface with the app
When user after choosing plan clicks proceed to payment then payment interface should open specifying amount user selected to pay and once payment is successful the user is re directed to successful registration notification  page.

influencer registration->influencer registartion form-> select plan->proceed to payment ->payment interface->(review page) create account->success

https://razorpay.com/docs/payments/server-integration/nodejs/payment-gateway/build-integration/

### PR DESCRIPTION
**Pull request by Hemu21**
## Description

Payment gatway is integrated for both web and mobile.

## Issue

[Issue Link](https://github.com/Kajol-Kumari/InfluMart/issues/90) resolve #90

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  @RohitGupta1237 can you please check

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Hemu21 why am i seeing kotlin and gradle files in this pr?

File: frontend/controller/paymentController.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  } else {
+    await handlePaymentMobile(order);
+  }
+};
+
+
+export const verifyPayment = async ({ paymentData }) => {
+  try {
+    const response = await fetch(
+      `${API_ENDPOINT}/api/payment/verify-payment`,
```
Comment (Kajol-Kumari) [inline frontend/controller/paymentController.js:58]: what is the `API_ENDPOINT` here?

File: frontend/screens/Influencer/PlanChooseInterface.js [hunk 2]
```diff
@@ -12,13 +12,33 @@ import { getSubscriptionPlans, subscribe } from "../../controller/subscriptionCo
...
+  const [orderId, setOrderId] = useState(null);
+
+  const initiatePayment = async () => {
+    console.log('Initiating payment...');
+    try {
+      const paymentData = await handlePayment();
+      const verificationResponse = await verifyPayment({ paymentData });
+      if (verificationResponse.message === 'Payment verified successfully') {
+        await handleSelectPlan()
+        console.log('Payment successful!');
```
Comment (Kajol-Kumari) [inline frontend/screens/Influencer/PlanChooseInterface.js:32]: can we add the required alerts for success and failure scenarios as well?

File: frontend/util/paymentFuntions.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+const handlePaymentWeb = async (order,showAlert) => {
+  const options = {
+    key: RAZORPAY_KEY_ID,
+    amount: "5000", // Amount in paise
+    currency: "INR",
+    name: "Influmart",
+    description: "Connects influencer with brands",
+    image: "https://i.imgur.com/3g7nmJC.png",
+    order_id: order.id,
+    prefill: {
```
Comment (Kajol-Kumari) [inline frontend/util/paymentFuntions.js:52]: these data is hardcoded again, is it expected?

File: frontend/util/paymentFuntions.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    modal: {
+      ondismiss: function () {
+        showAlert("Payment Cancelled","You cancelled to Procced Payment");
+      },
+    },
+  };
+
+  const success = await loadRazorpayScript();
+  if (!success) {
+    console.error("Razorpay SDK failed to load");
```
Comment (Kajol-Kumari) [inline frontend/util/paymentFuntions.js:69]: we will need alert here as well no?

**Pull request merged**